### PR TITLE
Export Default Webex Adapter

### PR DESCRIPTION
**Pull request by akoushke**
To trigger the release process.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,2 @@
 export {default as WebexSDKAdapter} from './WebexSDKAdapter';
+export {default as RoomsSDKAdapter} from './RoomsSDKAdapter';
```
Comment (adamweeks) [inline src/index.js]: Should `WebexSDKAdapter` be the default export?

That way one could:
```js
import WebexSDKAdapter from '@webex/sdk-component-adapter';
```

Or if someone was building their own adapter but wanted to use the RoomsSDKAdapter they could:

```js
import {RoomsSDKAdapter} from '@webex/sdk-component-adapter';

const myAdapter = {roomsAdapter: RoomsSDKAdapter};
```

Comment (lalli-flores) [reply src/index.js]: I like the idea of making `WebexSDKAdapter` a default export, but I still don’t think is a good idea to expose the sub-adapter.
My worry is that developers may use an “incomplete adapter” with components that might need more data that than of what is in the custom adapter wrapper.
This is why we use the “wrapper”. What sub-adapters a component uses is transparent, meaning, is not obvious/shown to users, so I feel not having a complete adapter will cause more harm than good.

Comment (adamweeks) [reply src/index.js]: Sounds good. We can revisit if we ever actually get that type of feature request. 

@akoushke can you update this to be the default export?

**Pull request merged**